### DISCUSSION TITLE
"Recommended based on people you follow" card text align

### DISCUSSION THREAD

**Discussion by mesaglio** [community/community#14301] [Other Feature Feedback, Questions, & Ideas] [answered]
There is text out of the card.
![image](https://user-images.githubusercontent.com/29051129/161660098-9ce8ccc8-ef46-4d99-8fe9-503405f08756.png)

**Comment by alex-kim-dev** [comment]
It overflows only on 1280px+ screen.
![screenshot_02](https://user-images.githubusercontent.com/45559664/161809093-3558722f-584e-4637-a403-a7255634d1b1.png)

**Comment by jamieshark** [comment, marked as answer]
This should be fixed as of today. Let us know if you're seeing differently or if you're seeing other related issues. Thanks!

**Reply by alex-kim-dev** [reply]
Thanks, this is now fixed, but now the cards are not aligned on the right side (on all screen widths):

<details><summary>Screenshot</summary>

![screenshot_01](https://user-images.githubusercontent.com/45559664/161918533-4112a8ea-7cd0-4796-97dd-4a50c9a090c5.png)
</details>

Not a big issue though.

**Reply by jamieshark** [reply]
thanks, we'll look into fixing this as well.
